### PR TITLE
Enise/simplenn withbase

### PR DESCRIPTION
**Pull request by eniseirem**
missing files readded 
> @ja-nina's nina/simple_nn SimpleNN and SimplerNN files 

reformated with basemodel
all models added to runfile

### CODE REVIEW

**Comment by ja-nina** [comment]
Woah! Ty so much for doing that!

**Comment by ja-nina** [comment]
Looks great

File: MammoNet/models/SimpleNN.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import torch.nn as nn
-from MammoNet.utils.utils import create_results_dir, setup_wandb, set_seed
-from MammoNet.global_variables import RESULTS_DIR, SEED
-from BaseModel import BaseModel
+from MammoNet.models.BaseModel import BaseModel
```
Comment (ja-nina) [inline MammoNet/models/SimpleNN.py:2]: Valid, valid, good finding!

**Review by ja-nina — APPROVED** [review]

**Comment by ja-nina** [comment]
This is more a note to future self, but we definitely need to add a suffix for the model - so that we can run them with and without augmentation and still get the saved files, but that's also an easy fix, and not for this pr

**Comment by eniseirem** [comment]
> This is more a note to future self, but we definitely need to add a suffix for the model - so that we can run them with and without augmentation and still get the saved files, but that's also an easy fix, and not for this pr

yeah you are totally right, was focused on finishing the code but that would create a confusion.

**Pull request merged**